src/core/shards/serialization.cpp

```diff
@@ -341,11 +341,110 @@ struct LoadImage : public FileBase {
   }
 };
 
+struct BytesToImage {
```

Comment:
I wouldn't name this BytesToImage, since ImageToBytes also exists but the can not be used in reverse looking this implementation:
- ImageToBytes outputs the raw pixel data into a byte variable
- BytesToImage reads any image using stbi_load (PNG/JPEG/other container formats, etc.)

Maybe merge is with LoadImage, in a similiar way that WritePNG now also has the ability to write to memory

Reply:
Good point, doing it now